### ISSUE TITLE
Options listed incorrectly in testlist_clm.xml

### ISSUE THREAD

**Issue by billsacks** [ESCOMP/ctsm#219]
The options in the v2-formatted testlist_clm.xml are listed incorrectly, so they aren't being picked up. I think the problem is that xml blocks like this:

```xml
      <machine name="cheyenne" compiler="intel" category="aux_clm"/>
      <options>
        <option name="wallclock">01:20:00</option>
        <option name="comment"  >Want a multi-year global crop restart test; this was 5 years when we were doing cold start, but 3 years is probably sufficient given that we have spun-up crop initial conditions</option>
      </options>
```

don't actually have the options nested under the machine. These should be changed to this:

```xml
      <machine name="cheyenne" compiler="intel" category="aux_clm">
        <options>
          <option name="wallclock">01:20:00</option>
          <option name="comment"  >Want a multi-year global crop restart test; this was 5 years when we were doing cold start, but 3 years is probably sufficient given that we have spun-up crop initial conditions</option>
        </options>
      </machine>

```

I think it's also allowable to put the options outside the machines block entirely, if you have options that should apply for all machines/compilers/categories.

**Comment by ekluzek** [comment]
Hmmm. Theres a cost if options have to be setup for specific machines, rather than all machines.They'll be a ton of duplication.  I thought I used the format so that options would apply to all machines. I based it off an example that was constructed that way.

Looking into the python testlist.py, it should work with this format:

```
<test ...>
  <options>
    <option name="wallclock">00:20</option>
  </options>
  ...
</test>
```
Whereas I nested it under machines instead of test. So I need to convert it to the above format.
